packages/panzer/mini-em/example/BlockPrec/main.cpp

```diff
@@ -238,9 +262,12 @@ int main_(Teuchos::CommandLineProcessor &clp, int argc,char * argv[])
       else
         xml = "solverAugmentationEpetra.xml";
     else if (solver == "MueLu-RefMaxwell") {
-      if (linAlgebra == "Tpetra")
-        xml = "solverMueLuRefMaxwell.xml";
-      else
+      if (linAlgebra == "Tpetra") {
+        if (dim == 3)
+          xml = "solverMueLuRefMaxwell.xml";
+        else
+          xml = "solverMueLuRefMaxwell2D.xml";
```

Comment:
Briefly, what are the main differences between these two xml files?

Reply:
For 2D problems, the B-field is discretized using discontinuous elements, so the face mass matrix is (block-)diagonal. We therefore use plain Jacobi to solve. (Using MueLu as in 3D can lead to errors, since all unknowns are detected as Dirichlet BC.)